typescript-selenium/mocha.opts

```diff
@@ -0,0 +1,15 @@
+#*********************************************************************
+ # Copyright (c) 2018 Red Hat, Inc.
```

Comment:
These white chars in the beginning of line cause `npm test` to fail with 
```
TypeError: arg.split is not a function
```

Reply:
Removed